Discovering/Property/UseCases/Property/AddPropertyForAdvisor/AddPropertyForAdvisorPresenter.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\RealEstate\Discovering\Property\UseCases\Property\AddPropertyForAdvisor;
+
+use App\RealEstate\Discovering\Property\Entities\Property\AdvisorId;
+use App\RealEstate\Discovering\Property\Entities\Property\Country;
+use App\RealEstate\Discovering\Property\Entities\Property\Exception\BusinessRulesViolationException;
+use App\RealEstate\Discovering\Property\Entities\Property\PropertyId;
+
+interface AddPropertyForAdvisorPresenter
+{
+    public function show(
+        PropertyId $propertyId,
+        AdvisorId $advisorId,
+        Country $country,
+        int $priceInCents
+    ): void;
+
+    public function showBusinessRulesViolations(BusinessRulesViolationException $exception): void;
```

Comment:
This method shouldn't be here. The only responsibility of a Presenter is to "present" a response. This can be moved into the AddPropertyForAdvisorResponse